### ISSUE TITLE
Cloud quailty decrease mode

### ISSUE THREAD

**Issue by FelipeThePilot** [ResetXPDR/MSFS2020_AutoFPS#72]
So I wanted to use cloud quality decreasment function with the tlod recovery mode. I set my target fps with lossless scaling to 142 with 5 sensitivity. And tlod recovery to 100 (I set max tlod to 120)

I expect the program to decrease my cloud quality if the tlod is below 100 so I can get stable 144 fps. And if I get over the 100 tlod, then the cloud quality will be set to ultra

This time the program decrease the cloud quality, but not that much that I want - only to medium not to low. Also when the tlod goes above 100 then the cloud quality is set to high, not to ultra as I want.

Do you have any solution for that? 

I mean, if you have time and you want to, then the best solution will be to create cloud mode in auto fps that will work the same as tlod and olod, because on the ground cloud quality make a big difference in fps and not that big visual difference, however on the cruise level it works other way around

**Comment by ResetXPDR** [comment]
Per the readme, if your initial cloud quality isn't set right:

- Do not change TLOD, OLOD and Cloud Quality MSFS settings manually while in a flight with this app running as it will conflict with what the app is managing and they will not restore to what you set when you exit your flight. If you wish to change the defaults for these MSFS settings, you must do so either without this app running or, if it is, only while you are in the MSFS main menu (ie not in a flight).

TLOD recovery is not the TLOD at which cloud quality reduction activates, it is the TLOD at which your cloud quality is restored.
Also per the readme:

- Decrease Cloud Quality - When enabled, will reduce/restore cloud quality by one level if the activation condition is met. 
  - TLOD (FPS Sensitivity and FPS Tolerance TLOD Automation Methods)
    - Decreases when TLOD has already auto reduced to TLOD Min and FPS is still below target FPS by more than the FPS tolerance.
    - Cloud Recovery TLOD with optional + (resultant TLOD must fall within TLOD Min+5 and TLOD Max-5)
      - The TLOD level required to cancel an active cloud quality reduction state and restore cloud quality back to its initial higher quality level.
      - Ideally set to 50 TLOD or more above TLOD Min to provide a TLOD buffer to minimise the chance that cloud quality will constantly change down and up.
      - When + is checked, Cloud Recovery TLOD becomes relative to TLOD Min instead of absolute.

Re having cloud mode work the same as TLOD and OLOD, the app is coded the way it is because only one item should be controlling the automation at a time, and you already have a choice of using TLOD or GPU load as the trigger method for cloud quality. This will not be changing.

In short, everything you ask about (except your proposed cloud mode solution) is in the readme, which you need to read more closely. Please read it in detail before asking any more questions and only then if your question is not covered there.
